config/default_records/submission_templates/011_scrna_core_cdna_prep_submission_templates.wip.yml

```diff
@@ -22,15 +22,15 @@ Limber-Htp - scRNA Core cDNA Prep Input:
     request_type_keys: ["limber_scrna_core_cdna_prep_input"]
     product_line_name: Short Read
     product_catalogue_name: scRNA Core
-Limber-Htp - scRNA Core Chromium HT 5p Aggregation:
+Limber-Htp - scRNA Core Chromium GEM-X 5p Aggregation:
   submission_class_name: "LinearSubmission"
   related_records:
-    request_type_keys: ["limber_scrna_core_chromium_ht_5p_aggregation"]
+    request_type_keys: ["limber_scrna_core_chromium_gem_x_5p_aggregation"]
     product_line_name: Short Read
     product_catalogue_name: scRNA Core
-Limber-Htp - scRNA Core Chromium HT:
+Limber-Htp - scRNA Core Chromium GEM-X 5p:
   submission_class_name: "LinearSubmission"
   related_records:
-    request_type_keys: ["limber_scrna_core_chromium_ht"]
+    request_type_keys: ["limber_scrna_core_chromium_gem_x_5p"]
```

Comment:
Is the addition of '5p' here intentional?

Reply:
I think using 5p is consistent in this case.